server/src/__tests__/executables.test.ts

```diff
@@ -0,0 +1,44 @@
+import * as path from 'path'
+import { Executables } from '../executables'
+
+const executables = Executables.fromPath(
+  path.resolve(__dirname, '..', '..', '..', 'testing', 'executables'),
+)
+
+describe('list', () => {
+  it('It finds executables on the PATH', () => {
+    expect.assertions(1)
+    const result = executables.then(ex => ex.list().find(x => x === 'iam-executable'))
+    return expect(result).resolves.toBeTruthy()
+  })
+
+  it.skip('It only considers files that have the executable bit set', () => {
```

Comment:
There is a skip here. 

Btw, you should remove the “It” from the message.

Reply:
@skovhus The `skip` is there because the test fails currently as it doesn't look at the executable bit yet  😅